authz.go

```diff
@@ -40,15 +40,14 @@ func PermissionAllowed(p Permission, ps []Permission) bool {
 	}
 
 	for _, perm := range ps {
-		permID := ID(0)
-		if perm.ID != nil {
-			permID = *perm.ID
-			if !permID.Valid() {
-				return false
+		if perm.ID == nil {
```

Comment:
What do you think of this to remove else:

```go
if perm.Action != p.Action || perm.Resource != p.Resource || perm.OrgID != p.OrgID {
    continue
}
```

Reply:
my intuition says keep the explicit if else in this case, but I'm open to switching it if you'd prefer to have it the other way.